aikau/src/main/resources/alfresco/services/SearchService.js

```diff
@@ -196,7 +199,7 @@ define(["dojo/_base/declare",
             }
 
             var sort = "";
-            if (payload.sortField != null && (payload.sortField === "null" || payload.sortField === ""))
+            if (payload.sortField && payload.sortField === "null")
```

Comment:
if it has to be equal to the string value `"null"` then why not just `if (payload.sortField === "null")`?